src/app/core/data/remote-data.ts

```diff
@@ -55,4 +55,8 @@ export class RemoteData<T> {
     return this.state === RemoteDataState.Success;
   }
 
+  get hasNoContent(): boolean {
+    return this.statusCode === 204;
```

Comment:
I wouldn't touch the way that works now. The way the remotedata state is determined will change with the PR for #739, and that code takes 204 in to account: there is a NoContent interface and in the case of a 204 a RemoteData<NoContent> with the state successful will be returned. That way a service method can put in its signature that it expects a 204 response.

Reply:
Since @artlowel notes this 204 response behavior will be cleaned up in a follow-up PR, I'm OK with the current approach as-is.  So, I'll mark this comment as "resolved".